custom_components/birthdays/__init__.py

```diff
@@ -33,10 +38,12 @@ async def async_setup(hass, config):
     devices = []
 
     for birthday_data in config[DOMAIN]:
-        name = birthday_data[CONF_NAME]
-        date_of_birth = birthday_data[CONF_DATE_OF_BIRTH]
-        icon = birthday_data[CONF_ICON]
-        devices.append(BirthdayEntity(name, date_of_birth, icon, hass))
+        unique_id            = birthday_data[CONF_UNIQUE_ID]
```

Comment:
Same comment here as lines 70-73, single space between variable and assignment (for instance `unique_id = birthday_data[CONF_UNIQUE_ID]`)